src/server/aws-ses/aws-ses.js

```diff
@@ -0,0 +1,67 @@
+// @flow
+import SES from 'aws-sdk/clients/ses'
+import conf from '../server.config'
+
+const accessKeyId = conf.awsSesAccessKey || null
```

Comment:
no need for '|| null'

Reply:
I think it is best to validate the input without regard if the input was already validated somewhere else in the code
keeping the component as independent and decoupled as possible